CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+# Contributing
+
+## Getting Started
+
+### Development setup
+
+Make sure you have Bun installed on your machine. If you don't have it, you can download it [here](https://bun.sh/docs/installation).
+
+If you can't install Bun, you can always use [Node.js](https://nodejs.org/en/) with the `npm` command instead, but it will not be as fast as Bun.
+
+First, install dependencies:
+
+```bash
+bun install
+```
+
+Then, run the development server:
+
+```bash
+bun dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+### Build
+
+When you build the project, you pre-render all the Server Side Generated (SSG) pages. This makes the site load faster and perform better and behave like it will when it is deployed. When serving the built project it will not hot reload when you make changes to the code like it does in development mode.
+
+You can build the project with the following command:
+
+```bash
+bun run build
+```
+
+Then setup environment variables by copying the `.env.example` file to `.env` and fill in the values. `.env` files are used to store sensitive information like API keys and database credentials and it will not be committed to the repository.
```

Comment:
This section is in the wrong place. It is probably my bad when I wrote it but it needs to happend when setting up the project after running `bun install`